src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -160,17 +160,23 @@ public void clickGroupByTeam() {
         submitEditForm();
     }
 
-    public void clickCollapseExpandButton() {
+    public void clickCollapseExpandButtonAndWaitForPanelsToExpand() {
         click(collapseExpandButton);
+        waitForPanelsToExpand();
     }
 
     public void expandPanels() {
         if (isElementPresent("collapse-panels-button")) {
-            clickCollapseExpandButton();
+            click(collapseExpandButton);
             waitForPanelsToExpand();
```

Comment:
Can use `clickCollapseExpandButtonAndWaitForPanelsToExpand()` here.

Reply:
rename `waitForPanelsToExpand` ?

Reply:
No just change the two statements (click, wait...) to one statement already defined above.